library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -481,12 +505,22 @@ impl Command {
             argv: *const *mut c_char,
             envp: *const *mut c_char,
         ) -> i32 {
-            let mut tries_left = MAX_FORKSPAWN_TRIES;
+            let mut minimum_delay = None;
```

Comment:
Same thought as before, except clearly we should hoist the `static` up to be visible to both of these functions.